helm/charts/infra/Chart.yaml

```diff
@@ -3,6 +3,6 @@ name: infra
 description: A Helm chart for Infra
 type: application
 ## {x-release-please-start-version}
-version: 0.14.1
-appVersion: 0.14.1
+version: 0.14.2
+appVersion: 0.14.2
```

Comment:
There's no need to increment the version. The helm chart version will be bumped automatically once a release is tagged and built.

Reply:
I do a clean helm install on a vanilla kubernetes cluster ( based on rancher RKE ). You should be able to replicate the issue by starting a clean helm install with a refresh PVC.